app/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Formatting

Reply:
That's strange; my formatter wants to do this for all XML files. Can you confirm what your formatter does for this file, @subsymbolic ?

Reply:
Interesting, mine doesn't change it.

Reply:
That's annoying. I've reverted the change, and we can sync up our formatting settings later.